app/assets/javascripts/discourse/app/components/composer-messages.js

```diff
@@ -183,6 +183,11 @@ export default class ComposerMessages extends Component {
       return;
     }
 
+    // We don't care about similar topics when creating with a form template
+    if (this.composer?.category?.form_template_ids.length > 0) {
```

Comment:
I suspect we want to hide all messages while in form template mode? If so, maybe a CSS solution is best?